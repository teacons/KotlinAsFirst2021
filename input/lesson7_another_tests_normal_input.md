Принимая во ~~внимание показатели успешности~~, внедрение современных методик выявляет срочную потребность существующих финансовых и административных условий.

Предварительные выводы неутешительны: начало ***~~повседневной~~*** работы по формированию позиции создаёт необходимость включения в производственный план целого ряда внеочередных мероприятий с учётом комплекса укрепления моральных ценностей.
Ра*знообразный и **богатый опыт гово~~рит нам, что новая модель орган**изационной деятельн~~ости является качестве*нно новой ступенью форм воздействия.

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do 
eiusmod tempor incididunt ut labore et dolore magna aliqua. 
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.



Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.




Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.