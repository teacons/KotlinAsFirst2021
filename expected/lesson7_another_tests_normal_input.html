<html>
<body>
<p>
    Принимая во <s>внимание показатели успешности</s>, внедрение современных методик выявляет срочную потребность существующих финансовых и административных условий.
</p>
<p>
    Предварительные выводы неутешительны: начало <b><i><s>повседневной</s></i></b> работы по формированию позиции создаёт необходимость включения в производственный план целого ряда внеочередных мероприятий с учётом комплекса укрепления моральных ценностей.
    Ра<i>знообразный и <b>богатый опыт гово<s>рит нам, что новая модель орган</b>изационной деятельн</s>ости является качестве</i>нно новой ступенью форм воздействия.
</p>
<p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
    eiusmod tempor incididunt ut labore et dolore magna aliqua.
    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
</p>
<p>
    Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
</p>
<p>
    Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
</p>
</body>
</html>